### PR TITLE
Add container image tag filters

### PR DESCRIPTION
**Pull request by adamlazik1**
In Managing Content guide, section 8.: expanded list to include container image tag filters.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/common/modules/con_managing-content-views.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {ProjectName} uses Content Views to create customized repositories from the repositories.
 To do this, you must define which repositories to use and then apply certain filters to the content.
-These filters include both package filters, package group filters, errata filters, and module stream filters.
+These filters include both package filters, package group filters, errata filters, module stream filters and container image tag filters.
```
Comment (Levi-Leah) [inline guides/common/modules/con_managing-content-views.adoc:6]:
```suggestion
These filters include both package filters, package group filters, errata filters, module stream filters, and container image tag filters.
```

Comment (adamlazik1) [reply guides/common/modules/con_managing-content-views.adoc:6]: Thanks for this!

**Comment by adamlazik1** [comment]
@maximiliankolb Could you take a look at this small change please?

File: guides/common/modules/con_managing-content-views.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 {ProjectName} uses Content Views to create customized repositories from the repositories.
 To do this, you must define which repositories to use and then apply certain filters to the content.
-These filters include both package filters, package group filters, errata filters, and module stream filters.
+These filters include both package filters, package group filters, errata filters, module stream filters, and container image tag filters.
```
Comment (maximiliankolb) [inline guides/common/modules/con_managing-content-views.adoc:6]:
```suggestion
These filters include package filters, package group filters, errata filters, module stream filters, and container image tag filters.
```

**Comment by adamlazik1** [comment]
@maximiliankolb Thank you for your suggestions. Incorporated the suggested change.

**Review by maximiliankolb — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: ✅ 
   746a6c475..aca78a4d5  3.4 -> 3.4
   5478573d5..1658b379b  3.3 -> 3.3
   2c9f5952c..c98cdff6e  3.2 -> 3.2
   b56df0759..38f8f5024  3.1 -> 3.1

**Pull request merged**